client/app/lib/styl/tooltip.styl

```diff
@@ -0,0 +1,61 @@
+$lightBg = white
+$lightColor = #727272
+
+div.kdtooltip.is-light
+  width                       250px
+  box-shadow                  0px 2px 10px 0px rgba(0,0,0,0.05)
+  font-size                   14px
+  letter-spacing              0.13px
+  line-height                 20px
+
+  div.wrapper
+    bg                        color, white
+    shadow                    0 10px 20px 5px rgba(0,0,0,0.15)
+    padding                   20px 20px 20px 20px
+
+  .copied-link
+    color                     $lightColor
+
+
+  .copied-link strong
+    font-weight               600
+
+  h2
+    color                     $lightColor
+    font-weight               600
+    letter-spacing            0.11px
+
+  h3
+    color                     #48BA7D
+    font-weight               600
+
+  p
+    color                     $lightColor
+    margin-top                10px
+
+  &.placement-top
+    &:after
+      border-top-color        $lightBg
+
+  &.placement-bottom
+    &:after
+      border-bottom-color     #fff
+
+  &.placement-left
+    &:after
+      border-left-color       #fff
+
+  &.placement-right
+    &:after
+      border-right-color      #fff
+
+
+  &.just-text
+
+    div.wrapper
+      shadow                  0 3px 7px rgba(255, 255, 255, 0.25)
+      bg                      color, white
+      color                   #727272
+
+      font-size               12px
+      font-weight             300
```

Comment:
Once terminal is opened, this is gonna affect all the tooltips in the app. We should namespace this, or make sure every other tooltip looks good!